rest_framework_reactive/models.py

```diff
@@ -1,66 +1,71 @@
-from __future__ import absolute_import, division, print_function, unicode_literals
+from django.contrib.postgres.fields import JSONField
+from django.db import models, transaction
 
-from django import dispatch
-from django.db import transaction
-from django.db.models import signals as model_signals
+class Observer(models.Model):
+    """State for an observer."""
 
-from . import client
+    # Observer status.
+    STATUS_NEW = 'new'
+    STATUS_OBSERVING = 'observing'
+    STATUS_STOPPED = 'stopped'
 
-# Setup model notifications.
-observer_client = client.QueryObserverClient()
+    STATUS_CHOICES = (
+        (STATUS_NEW, "New"),
+        (STATUS_OBSERVING, "Observing"),
+        (STATUS_STOPPED, "Stopped"),
+    )
 
+    id = models.CharField(primary_key=True, max_length=64)
+    status = models.CharField(max_length=15, choices=STATUS_CHOICES, default=STATUS_NEW)
+    request = models.BinaryField()
+    last_evaluation = models.DateTimeField(auto_now_add=True)
+    poll_interval = models.IntegerField(null=True)
+    subscribers = models.ManyToManyField('Subscriber')
 
-@dispatch.receiver(model_signals.post_save)
-def model_post_save(sender, instance, created=False, **kwargs):
-    """
-    Signal emitted after any model is saved via Django ORM.
+    def __repr__(self):
```

Comment:
You can define `__str__` as:

    def __str__(self):
        return 'id={id}'.format(id=self.id)

and [default repr](https://github.com/django/django/blob/stable/1.11.x/django/db/models/base.py#L593) wraps it into `<Observer: >`

Reply:
Nice, done.